rpm/pext.spec

```diff
@@ -0,0 +1,44 @@
+#Disable debug packages
+%define debug_package %{nil}
+
+Name:	pext
+Version:	0.9
+Release:	1%{?dist}
+Summary:	Python-based extendable tool
+
+License:	GPLv3
+URL:		https://github.com/Pext/Pext
+Source0: 	https://github.com/Pext/Pext/archive/v%{version}.zip
+#This removes the dependency since dependency checks are done by dnf/yum/rpm.
+#Also in Fedora the module name is PyQt5
+Patch0:		remove-dependency.patch
+
+BuildRequires:	python3-devel
```

Comment:
Wouldn't this need python3-pygit2, because of the pygit2 import in setup.py and attempts to get the version info from git? Maybe that should be patched out of setup.py instead?